### DISCUSSION TITLE
Walletconnect project - Localhost as allowed domain

### DISCUSSION THREAD

**Discussion by luarx** [WalletConnect/walletconnect-monorepo#1672] [Developer Support]
Hi everyone!

Could it be possible to use **localhost** and **localhost:*** as allowed domain?
We think that it could be useful to do some test in our local environments :)

![image](https://user-images.githubusercontent.com/22997139/203968301-1af81a71-9e50-4998-877f-28b254c4902b.png)

Thanks in advance!

**Comment by heilhead** [comment]
Hi @luarx! Currently, origin validation is skipped if the origin's hostname is either `localhost` or `127.0.0.1`. Works for all protocols (e.g. `http://`, `https://` etc) and ports.

**Comment by luarx** [comment]
Great! Thanks a lot for the clarification :D
Good to know that you had already thought about that!

**Comment by dev-bjoern** [comment]
Does not work localhost with subdomains does it?

We have something like: testing.localhost and connections are rejected.

@heilhead Would it be possible to allow these as well?
